### ISSUE TITLE
Use system openblas also on Fedora-27

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage#29393]
<div id="comment:0"></div>

Original ticket description 

(a) Fedora does not install openblas.pc, so spkg-configure.m4 does not find it. 

    ---> Resolved by #29398 **except for `fedora-27` (see discussion)**

(b) Also, (as noted in #29361) when installing our openblas on fedora-32-standard (which has openblas installed), openblas-dependent packages such as iml, gsl fail to compile. This is in contrast to fedora-32-minimal (which has no system openblas) - where iml builds successfully

    ---> Resolved by #29378

Depends on #29398

CC:  @dimpase

Component: **build: configure**

_Issue created by migration from https://trac.sagemath.org/ticket/29393_

**Comment by mkoeppe** [comment]
Description changed:
``````diff
--- 
+++ 
@@ -1,4 +1,9 @@
-Fedora does not install openblas.pc, so spkg-configure.m4 does not find it
+Original ticket description 
 
-Also, (as noted in #29361) when installing our openblas on fedora-32-standard (which has openblas installed), openblas-dependent packages such as iml, gsl fail to compile. This is in contrast to fedora-32-minimal (which has no system openblas) - where iml builds successfully
+(a) Fedora does not install openblas.pc, so spkg-configure.m4 does not find it. 
 
+    ---> Resolved by #29398 **except for `fedora-27` (see discussion)**
+
+(b) Also, (as noted in #29361) when installing our openblas on fedora-32-standard (which has openblas installed), openblas-dependent packages such as iml, gsl fail to compile. This is in contrast to fedora-32-minimal (which has no system openblas) - where iml builds successfully
+
+    ---> Resolved by #29378
``````

**Comment by mkoeppe** [comment]
outdated
